### PR TITLE
chore: Add presets of colors for Theme Setttings

### LINKED ISSUE #32430: chore: WDS update recommended colors in theme color picker
Different sets for light and dark modes.

### PR DESCRIPTION
**Pull request by jsartisan**
Fixes #32430

## Summary by CodeRabbit

- **New Features**
  - Added color mode options (Dark and Light) and color presets in the theme settings.
  - Updated the color picker UI and functionality in theme property settings.
- **Enhancements**
  - Reorganized the layout of the App Settings to improve user navigation.
  - Enhanced theme customization options to include new section headers and informational cards.
- **Refactor**
  - Updated various components to use new and more efficient components and imports, improving performance and user experience.

### CODE REVIEW

**Comment by jsartisan** [comment]
@ichik can you try this branch on your local once?

File: app/client/src/pages/Editor/AppSettingsPane/AppSettings/index.tsx [hunk 1]
```diff
@@ -237,21 +237,22 @@ function AppSettings() {
...
-                    {isAnvilLayout ? (
-                      <WDSThemePropertyPane />
-                    ) : (
+                <ThemeContentWrapper>
+                  {isAnvilLayout ? (
+                    <WDSThemePropertyPane />
+                  ) : (
+                    <>
+                      <div className="px-4">
+                        <SectionTitle className="flex items-center gap-2">
```
Comment (marks0351) [inline app/client/src/pages/Editor/AppSettingsPane/AppSettings/index.tsx:246]: we can move the tile and beta card into ThemePropertyPane right?

File: app/client/src/pages/Editor/WDSThemePropertyPane/index.tsx [hunk 2]
```diff
@@ -79,37 +83,83 @@ function WDSThemePropertyPane() {
...
-    <main className="mt-1">
+    <main className={styles.main}>
       {/* COLORS */}
       <SettingSection className="px-4 pb-3" isDefaultOpen title="Color">
         <section className="space-y-2">
-          <ColorPickerComponent
-            changeColor={(color: string) => {
+          <SegmentedControl
+            isFullWidth
+            onChange={(value: string) => {
```
Comment (marks0351) [inline app/client/src/pages/Editor/WDSThemePropertyPane/index.tsx:115]: can we declare these arrow functions?

**Review by marks0351 — CHANGES_REQUESTED** [review]

**Comment by ichik** [comment]
@jsartisan looks good. A few notes:
1. Light should be on the left side of segmented control, dark on the right
2. Switching the seed using the swatches doesn't update the code in the input until the panel is refreshed (i.e. closing it and opening it again)
3. The full/native color picker seems to always open from `#000` as a starting point which looks fine with Chrome color picker but can be a bit too much for macOS native color picker (possibly for some others too). If we could set the starting point to whatever seed is currently set that'd be a much nicer experience (see screenshots from FF and Chrome).
image
image

**Review by ichik — APPROVED** [review]

**Review by marks0351 — APPROVED** [review]

File: app/client/src/pages/Editor/ThemePropertyPane/index.tsx [hunk 3]
```diff
@@ -31,7 +34,17 @@ export function ThemePropertyPane() {
     }
   }, [themingMode]);
 
-  return <div className="relative">{propertyPane}</div>;
+  return (
+    <>
+      <div className="px-4">
```
Comment (KelvinOm) [inline app/client/src/pages/Editor/ThemePropertyPane/index.tsx:39]: You don't need this div. You can add `px-4` class to `SectionTitle`

**Review by KelvinOm — APPROVED** [review]

**Pull request merged**